mne/io/array/tests/test_array.py

```diff
@@ -71,7 +71,7 @@ def test_array_copy():
 
 
 @pytest.mark.slowtest
-@requires_version('scipy', '0.12')
+@requires_version('scipy', '1.2.0')
```

Comment:
@hoechenberger you should not have to bump requirements for an option that is not specifically tested here. In other words scipy1.2 should only become a requirement if you set average different from 'mean' is the psd_welch function.

Reply:
I had to bump requirements for all tests that invoke `psd_welch()` as my current implementation would _always_ pass the `average` keyword down to `scipy.signal.welch()`. This was probably not the best idea. I will revert adding all these `requires_version` decorators in ad3063a, and add special handling for `average='mean'` to retain backward-compat, i.e., will do sth that effectively boils down to this:
```python
if average == 'mean':
    scipy.signal.welch(...)
elif average == 'median':
    scipy.signal.welch(, average='median')
```